backend/Handlers.go

```diff
@@ -109,6 +110,22 @@ func VerifyUser(w http.ResponseWriter, r *http.Request) {
 		fmt.Println("Sending failed token exchange error")
 	}
 
+	users, err := fbClient.Ref("/users")
+	if err != nil {
+		respondWithError(w, FailedDBCall, err.Error())
+		fmt.Println("Failed to save user profile")
+	}
+	person := Person{
+		ID:          lip.ID,
+		Firstname:   lip.FirstName,
+		Lastname:    lip.LastName,
+		AccessToken: aTokenResp,
+		LiProfile:   lip,
+	}
+	addUser := make(map[string]interface{})
+	addUser[lip.ID] = person
+	defer users.Update(addUser)
```

Comment:
Will this override the Firebase object or just add a new key-value pair?

Reply:
Talked offline. It updates if exists and creates if not exists